inferno-ml-server-types/src/Inferno/ML/Server/Types.hs

```diff
@@ -703,6 +704,7 @@ data IValue
   | ITuple (IValue, IValue)
   | ITime EpochTime
   | IEmpty
+  | IArray (Vector IValue)
```

Comment:
Is `IValue` the type of all values that can be stored in input/output PIDs in TachDB? I don't think they support arrays, if so.